examples/data-sources/env0_gcp_oidc_credentials/data-source.tf

```diff
@@ -0,0 +1,15 @@
+resource "env0_azure_oidc_credentials" "credentials" {
```

Comment:
forgot to add an example for azure oidc... added now...

Reply:
this is the gcp oidc credential file

Reply:
too much copy and paste (:
I'll fix it.